panda/src/windisplay/winGraphicsWindow.cxx

```diff
@@ -3072,39 +3074,54 @@ get_icon(const Filename &filename) {
   }
 
   // If it wasn't found, resolve the filename and search for that.
-
-  // Since we have to use a Windows call to load the image from a filename, we
-  // can't load a virtual file and we can't use the virtual file system.
   Filename resolved = filename;
-  if (!resolved.resolve_filename(get_model_path())) {
-    // The filename doesn't exist along the search path.
-    if (resolved.is_fully_qualified() && resolved.exists()) {
-      // But it does exist locally, so accept it.
+  if ((resolved.resolve_filename(get_model_path())) || (resolved.is_fully_qualified() && resolved.exists())) {
```

Comment:
It may still be in the VFS if `exists()` returns true if there is an overlapping mount.

Here's how to get the system filename from a VFS filename:
```python
      PT(VirtualFile) vfile = ...;
      // check vfile
      SubfileInfo info;
      if (vfile->get_system_info(info) && info.get_start() == 0) {
        Filename fn = info.get_filename();
        // do something with fn.to_os_specific()
```